package.json

```diff
@@ -33,7 +33,7 @@
     "AddThis"
   ],
   "peerDependencies": {
-    "react": ">=0.13.2"
```

Comment:
Why remove this?

Reply:
@jhorwit2 I'd assumed this was left over from when fluxthis pulled in React's proptype library - now that it uses the `proptypes` module, I couldn't see any direct dependency of fluxthis on react.